_config.yml

```diff
@@ -17,10 +17,11 @@ description:
   vi: Giáo trình cho ngôn ngữ lập trình Elixir
   ru: Уроки программирования на языке Elixir
   sk: Lekcie o programovacom jazyku Elixir
-  fr: Leçons sur le langage de programmation Elixir
+  gr: Μαθήματα προγραμματισμού της γλώσσας Elixir
+  fr: Lessons about the Elixir programming language
```

Comment:
Hi @Kosmas, could you correct this change?  Looks like the French translation is getting removed.